xcat-inventory/xcclient/allien/resources/auth.py

```diff
@@ -0,0 +1,75 @@
+###############################################################################
+# IBM(c) 2019 EPL license http://www.eclipse.org/legal/epl-v10.html
+###############################################################################
+
+# -*- coding: utf-8 -*-
+
+from flask import request, current_app, make_response, jsonify
+from flask_restplus import Resource, Namespace
+import uuid
+from exceptions import *
+from xcclient.xcatd.client.xcat_exceptions import XCATClientError
+from ..invmanager import *
+from . import auth_request
+
+ns = Namespace('auth', description='The Authorization section for APIs')
+@ns.route('/login')
+class ToLogin(Resource):
+
+    def get(self, code=200):
+        return make_response('''
+        <form action="" method="post">
+            <p><input type=text name=username>
+            <p><input type=password name=password>
+            <p><input type=submit value=Login>
+        </form>
+        ''', code)
+    def post(self):
+        try:
+            if request.headers['Content-Type'] == 'application/json':
+                post_data = request.get_json()
+            elif 'form' in request.headers['Content-Type']:
+                post_data = request.form.to_dict()
+            else:
+                return self.get(400)
+        except Exception:
+            return ns.abort(400)
+        usr = post_data['username']
+        pwd = post_data['password']
+        if check_user_account(usr, pwd, usertype="xcat-user"): 
```

Comment:
We have two types of user are supported,  `xcat` and `xcat-user`

Reply:
Yeah, now, with list all user account with key=xcat and key=xcat-user